### DISCUSSION TITLE
Why was the `ui` option in config marked as internal?

### DISCUSSION THREAD

**Discussion by posva** [vitest-dev/vitest#4737] [Q&A] [answered]
In the past, I was using the `ui` option in the test config to enable the UI, but this stopped working and now you need to pass the `--ui` flag. I noticed that the option was marked as internal. What I find confusing is that the option is there, can be set to `true` and if it is set to true, the CLI will log that the UI is running while it's not:

- https://stackblitz.com/edit/vitest-dev-vitest-ubuxgt?file=vite.config.ts,package.json&initialPath=__vitest__/

I was wondering if this is normal behavior. If it is, why is `ui` still an option? shouldn't it be removed altogether? I know it's marked as internal but isn't it some kind of context that is only enabled with `--ui`?

**Comment by sheremet-va** [comment, marked as answer]
I guess it was by accident. `@internal` flag was there from the beginning, but it had `WIP` string there. When we updated eslint, it found this and removed the comment.

**Reply by posva** [reply]
Thanks for the explanation 🙌
I don't know enough about Vitest's internals to have a solid opinion. Still, I know people don't read the comments of the options, so hiding it from the config would prevent a few users from falling into the trap.

**Reply by sheremet-va** [reply]
I think we can safely remove the internal flag at this point.
